doc/nrf/ug_radio_fem.rst

```diff
@@ -73,6 +73,12 @@ The following FEM implementations are supported:
 * The nRF21540 2-pin simple GPIO implementation.
   To use it, set the :kconfig:option:`CONFIG_MPSL_FEM_SIMPLE_GPIO` Kconfig option to ``y``.
 
+It is possible to only use the :ref:`nrfxlib:mpsl_fem` API if your application does not require other MPSL features.
+This might be useful when you want to run simple radio protocols that are not intended to be used concurrently with other protocols.
```

Comment:
```suggestion
This could be useful when you want to run simple radio protocols that are not intended to be used concurrently with other protocols.
```